wolfcrypt/src/wc_port.c

```diff
@@ -1174,6 +1174,171 @@ int wolfSSL_CryptHwMutexUnLock(void) {
 
 #endif
 
+#if defined(_WIN32_WCE)
```

Comment:
This block of time code was surrounded by an "#ifndef NO_ASN_TIME" block when it was in ./wolfcrypt/src/asn.c.  It should be inside the same define here for users wanting to eliminate time code completely.